infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -232,7 +231,7 @@ kuberay-operator:
 # Kuberay API Server
 # ===================
 
-kuberayApiServerEnable: true
+kuberayApiServerEnable: false
```

Comment:
is there anything still using the api server? if not, may as well rip off the bandaid and cut all this stuff out now :smile:

Reply:
No, nothing is using the api server.  It can be taken off along with the raycluster.